### ISSUE TITLE
Default value is double quoted if string

### ISSUE THREAD

**Issue by FlorianLudwig** [koxudaxi/datamodel-code-generator#158]
**Describe the bug**
Default value is double quoted if string

**To Reproduce**

Example schema:
```json

{
    "definitions": {
        "io.k8s.api.core.v1.ComponentStatus": {
            "description": "ComponentStatus (and ComponentStatusList) holds the cluster validation info.",
            "properties": {
                "apiVersion": {
                    "description": "APIVersion defines the versioned schema of this representation of an object. Servers should convert recognized schemas to the latest internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources",
                    "type": "string",
                    "default": "v1"
                },
                "kind": {
                    "description": "Kind is a string value representing the REST resource this object represents. Servers may infer this from the endpoint the client submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds",
                    "type": "string",
                    "enum": [
                        "ComponentStatus"
                    ],
                    "default": "ComponentStatus"
                }
            },
            "type": "object",
            "x-kubernetes-group-version-kind": [
                {
                    "group": "",
                    "kind": "ComponentStatus",
                    "version": "v1"
                }
            ]
        }
    }
}

```

Used commandline:
```
$ datamodel-codegen --input k8s_1.16.4_patched.json --input-file-type jsonschema --output test_
```

**Expected behavior**

```python
class ComponentStatus(BaseModel):
    apiVersion: Optional[str] = Field(
        "v1",
        description="...",
    )
```

But got:
```python
class ComponentStatus(BaseModel):
    apiVersion: Optional[str] = Field(
        "'v1'",
        description="...",
    )
```

**Version:**
 - OS: Linux
 - Python Verison: 3.8
 - datamodel-code-generator git master, 2cc4d0d17ed5a1ed249c06bdc6d4d2dabbe34f71

**Additional context**
This is a regression, was working a few commits ago.

**Comment by koxudaxi** [comment]
> This is a regression, was working a few commits ago.

Sorry, I had to add tests.

**Comment by koxudaxi** [comment]
@FlorianLudwig 
I have released it as version `0.5.10`
